DeezyMatch/utils.py

```diff
@@ -359,6 +359,12 @@ def read_command_candidate_ranker():
     parser.add_argument("-sz", "--search_size",
                         help="search size", default=4)
 
+    parser.add_argument("-ld", "--length_diff",
+                        help="max length difference", default=120)
```

Comment:
I think the default value of `length_diff` should be None? (based on the function args above)

Reply:
True! Changed [here](https://github.com/Living-with-machines/DeezyMatch/pull/121/commits/daa3ee6d214bcabd081d0e3cedb42b7387a78955#diff-3ceb82fa8065786fddf396bba8322edc2c0fd225ded196860dd4e897af3e48a8).